### PR TITLE
Core 2.3.4 to master

### PR DESCRIPTION
**Pull request by leslietilling**
## Purpose of this pull request

Publish 2.3.4 content updates

## Affected Magento editions

- [X] Open Source
- [X] Commerce
- [X] B2B

## Additional information

PRs to core-2.3.4 reviewed in private repository prior to 1/10/20 and additional PRs in this repository:

#234 #245 #272 #277 #280 #282 #284

Current builds on beta site

### CODE REVIEW

**Comment by leslietilling** [comment]
running tests

File: src/cms/adobe-stock-manage.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+1. Click <span class="btn">Search Adobe Stock</span>.
+
+{:.procedure}
+To search Adobe Stock assets and add a stock image to a category:
+
+1. On the _Admin_ sidebar, go to **Catalog** > **Categories**.
+
+1. Click **Add Root Category** or **Add Subcategory**.
+
+   If you want to add the image to an existing category, click the category name in the list on the left..
```
Comment (hickskenh) [inline src/cms/adobe-stock-manage.md]:
```suggestion
   If you want to add the image to an existing category, click the category name in the list on the left.
```

File: src/cms/adobe-stock-manage.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+1. [Run a keyword search for Adobe Stock assets][search-by-keywords].
+
+1. Click <span class="btn">Filters</span>.
+
+    There are several filters available to refine your search results:
+
+    |Filter|Description|
+    |---|---|
+    |Subcategory|Filter for images that are **Photos** or **Illustrations**|
+    |Orientation|Filter for images by size, shape and aspect|
```
Comment (hickskenh) [inline src/cms/adobe-stock-manage.md]: all of this looks bold, is it a glitch?   i see the ** are closed.

Comment (hickskenh) [inline src/cms/adobe-stock-manage.md]:
```suggestion
    |Orientation|Filter for images by size, shape, and aspect|
```

File: src/cms/adobe-stock-manage.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+    - Related images, such as images from the same **series** or **model**
+    - Action buttons, such as [**Save Preview**][save-preview] and [**Save (and optionally license) Image**][save-licensed]
+
+    ![Adobe Stock image details]({% link images/images/adobe-stock/image-details.png %}){: .zoom}
+    _Adobe Stock image details_
+
+## Log In to Your Adobe Account
+
+To gain complete access to an image and eliminate the Adobe Stock watermark, you must [sign in with an Adobe account][adobe-signin] and purchase credits to
+license rights to use an image.
```
Comment (hickskenh) [inline src/cms/adobe-stock-manage.md]: is there a soft return here?

File: src/cms/adobe-stock-manage.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+
+All licensed images associated to your Adobe account will have an additional label displayed on them, making it clear which images you have licensed.
+
+![Adobe Stock search results with licensed images]({% link images/images/adobe-stock/licensed-images.png %}){: .zoom}
+_Search Results with Licensed Images_
+
+### Save Images to the Media Storage
+
+Images searched for via the Adobe Stock Integration can be saved to the Magento [Media Storage][media-storage] for easy re-use across your Magento
+store.
```
Comment (hickskenh) [inline src/cms/adobe-stock-manage.md]: soft return?

File: src/cms/adobe-stock-manage.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+
+You can save two types of images: an [image preview][save-preview] or a [licensed image][save-licensed].
+
+#### Save an Image Preview
+
+An image preview is a watermarked version of an Adobe Stock asset. Image previews are free and are a good way to experiment with different images before you
+decide to purchase a license for specific images and use them on your production stores.
```
Comment (hickskenh) [inline src/cms/adobe-stock-manage.md]: soft return?

File: src/cms/adobe-stock-manage.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+    _Save the preview image_
+
+1. Click <span class="btn">Confirm</span>.
+
+    The page redirects to the [Media Storage][media-storage] and your saved preview is displayed.
+
+#### Save a Licensed Image
+
+Adobe Stock assets that you want to use for your production Magento stores should be licensed to ensure you have legal access to the image as well as to
+eliminate the Adobe Stock watermark that is present on all [image previews][save-preview]. To license images or to save already-licensed images, you must be logged in to your Adobe account.
```
Comment (hickskenh) [inline src/cms/adobe-stock-manage.md]: soft return?

File: src/cms/adobe-stock.md [hunk 7]
```diff
@@ -0,0 +1,91 @@
+---
+title: Adobe Stock Integration
+---
+
+Merchants can integrate [Adobe Stock][adobe-stock] with Magento giving them access to countless media assets for use in their stores.
+
+![Adobe Stock Search Results]({% link images/images/adobe-stock/search-grid.png %}){: .zoom}
+
+The Adobe Stock service provides businesses with access to millions of high-quality, curated, royalty-free photos, vectors, illustrations, videos, templates, and 3D assets for all their creative projects. Magento users are able to quickly find, preview, and license Adobe Stock assets. Users can also save them to the
+[Media Storage][media-storage], all without leaving the Magento workspace.
```
Comment (hickskenh) [inline src/cms/adobe-stock.md]: soft return?

File: src/cms/adobe-stock.md [hunk 8]
```diff
@@ -0,0 +1,91 @@
...
+
+## Prerequisites
+
+This integration requires an [adobe.io Console][io-console] account and Magento 2.3.4 or newer. In order to license [Adobe Stock][adobe-stock] images, a paid [Adobe Stock][adobe-stock] plan and an [Adobe account][adobe-signin] are also required.
+
+## Integrate Magento and Adobe Stock
+
+Configuring the Adobe Stock Integration for Magento is a two-step process: [create an adobe.io integration][create-integration] to generate an API Key, and
+then [configure the Adobe Stock Integration in your Magento Admin][configure].
```
Comment (hickskenh) [inline src/cms/adobe-stock.md]: soft return?

File: src/cms/adobe-stock.md [hunk 9]
```diff
@@ -0,0 +1,91 @@
...
+
+### Create an Adobe.io integration
+
+1. Navigate to the [adobe.io Console][io-console] and under **Quick Start** click **Create integration**.
```
Comment (hickskenh) [inline src/cms/adobe-stock.md]:
```suggestion
1. Navigate to the [adobe.io Console][io-console] and under **Quick Start**, click **Create integration**.
```

File: src/cms/adobe-stock.md [hunk 10]
```diff
@@ -0,0 +1,91 @@
...
+
+1. Ensure **Access an API** is selected and click **Continue**.
+
+1. Under the **Creative Cloud** column, ensure **Creative SDK** is selected and click **Continue**.
+
+   At this time, _do not_ use the **Adobe Stock** service on this screen.
+
+1. On the final **Integration Details** screen, do the following:
+
+    - For **Name**, enter a descriptive name for your integration
```
Comment (hickskenh) [inline src/cms/adobe-stock.md]:
```suggestion
    - For **Name**, enter a descriptive name for your integration.
```

File: src/cms/adobe-stock.md [hunk 11]
```diff
@@ -0,0 +1,91 @@
...
+
+    - For **Description**, enter a longer form description for your integration
```
Comment (hickskenh) [inline src/cms/adobe-stock.md]:
```suggestion
    - For **Description**, enter a longer form description for your integration.
```

File: src/cms/editor-insert-image.md [hunk 12]
```diff
@@ -37,3 +37,10 @@ Use this method to insert an image that is available online, but resides on anot
     - In the **Title** field, enter a relevant title for the image.
 
 1. Click <span class="btn">Insert</span> to complete the process.
+
+## Method 3: Use the Adobe Stock Integration
+
+If you have an [Adobe Stock][adobe-stock] plan associated to your Adobe account, you can search for, license and save from the millions of available assets provided by
+the Stock service. See [Using Adobe Stock Images]({% link cms/adobe-stock-manage.md %}).
```
Comment (hickskenh) [inline src/cms/editor-insert-image.md]: soft return?

File: src/cms/page-builder-add-content-products.md [hunk 13]
```diff
@@ -20,75 +35,112 @@ _Product List In Storefront_
...
+
+   ![Product Appearances]({% link images/images-ee/page-builder-products-appearances.png %}){: .zoom}
+   _Product Appearances_
+
+1. In the **Carousel Settings**, configure the settings to match the following screenshot:
+
+   ![Carousel Settings]({% link images/images-ee/page-builder-products-carousel-settings.png %}){: .zoom}
+   _Carousel Settings_
+
+   See the [Product Carousel Settings] table below for descriptions of each setting.
```
Comment (hickskenh) [inline src/cms/page-builder-add-content-products.md]: looks like a link, but nothing is linked

File: src/cms/page-builder-add-content-products.md [hunk 14]
```diff
@@ -20,75 +35,112 @@ _Product List In Storefront_
...
+
+### Select Products By
+
+You can select your products by **Category**, **SKU**, or **Condition**. The Category and SKU settings can also be set within the Condition settings. However, selecting the Category or SKU buttons provides an additional **Sort By** option of **Position**. For the Category option, sorting by Position displays the products in the same order that they appear in your Catalog. For the SKU option, sorting by Position displays the products in the order they appear within the Product SKUs textbox. For more information on the Position setting, see [Select Products By Settings]: Product SKUs and Sort By.
```
Comment (hickskenh) [inline src/cms/page-builder-add-content-products.md]: in last sentence, is that a link to something?

File: src/cms/page-builder-add-content-products.md [hunk 15]
```diff
@@ -20,75 +35,112 @@ _Product List In Storefront_
...
-   - To complete the _Category is ..._ part of the condition, click **More** (...). Then, click the **Chooser** (![]({% link images/images/btn-chooser.png %}){: .Inline_larger}) after the field.
+   ![Product Sorting Options]({% link images/images-ee/page-builder-products-sortby.png %}){: .zoom}
+   _Product Sorting Options_
 
-      ![]({% link images/images-ee/page-builder-add-content-products-settings-condition-category-is.png %}){: .zoom}
-      _Condition_
+1. Enter the **Number of Products to Display** in the Product Carousel or Grid.
 
-   - In the category tree, drill down to the **Women & Tops** category and select the **Tees** checkbox.
+   Values can be from 1 to 999.
```
Comment (hickskenh) [inline src/cms/page-builder-add-content-products.md]:
```suggestion
   Values can be from `1` to `999`.
```

File: src/cms/page-builder-media-image.md [hunk 16]
```diff
@@ -17,32 +19,36 @@ _Image In Column_
...
+
+        - Navigate to the image and click the thumbnail. Then, click <span class="btn">Add Selected</span>.
 
-        ![]({% link images/images-ee/page-builder-media-image-gallery-add-selected.png %}){: .zoom}
-        _Add Selected Image_
+          ![]({% link images/images-ee/page-builder-media-image-gallery-add-selected.png %}){: .zoom}
+          _Add Selected Image_
 
-        The image appears in the target container at the placeholder location. Unlike a background image, you can move the image to a different position within the current container, or to a different container.
+      The image appears in the target container at the placeholder location. Unlike a background image, you can move the image to a different position within the current container, or to a different container.
```
Comment (hickskenh) [inline src/cms/page-builder-media-image.md]:
```suggestion
      The image appears in the target container at the placeholder location. Unlike a background image, you can move the image to a different position within the current container or to a different container.
```

File: src/cms/page-builder-media-slider.md [hunk 17]
```diff
@@ -442,13 +442,15 @@ If you want a title above the slider, simply add a Text content type above the s
...
-|Minimum Height|Specifies the minimum height of the slider in pixels.|
+| Setting        | Description                                                                                                                                                                                                                                                                                                              |
+|----------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Autoplay       | Determines if the slideshow begins automatically when the page loads. Options: Yes - The slider and its automated properties become active as soon as the page loads. No - After the first slide, which appears by default, the slide navigation (dots or arrows) must be clicked to display the next slide in sequence. |
```
Comment (hickskenh) [inline src/cms/page-builder-media-slider.md]:
```suggestion
| Autoplay       | Determines if the slideshow begins automatically when the page loads. Options: <br/>**Yes** - The slider and its automated properties become active as soon as the page loads. <br/>**No** - After the first slide, which appears by default, the slide navigation (dots or arrows) must be clicked to display the next slide in sequence. |
```

File: src/cms/page-builder-media-slider.md [hunk 18]
```diff
@@ -442,13 +442,15 @@ If you want a title above the slider, simply add a Text content type above the s
...
+| Autoplay Speed | Specifies the delay in milliseconds between each slide when Autoplay is used. Default value: 4000 ms (4 seconds)                                                                                                                                                                                                         |
+| Fade           | Determines if a fade effect is used to smooth the transition from one slide to the next. Options: Yes - The current slide has a fade-out effect before the next slide fades in to view. No - There is no special effect during the transition from one slide to the next.                                                |
```
Comment (hickskenh) [inline src/cms/page-builder-media-slider.md]:
```suggestion
| Fade           | Determines if a fade effect is used to smooth the transition from one slide to the next. Options:<br/>**Yes** - The current slide has a fade-out effect before the next slide fades in to view.<br/>**No** - There is no special effect during the transition from one slide to the next.                                                |
```

File: src/cms/page-builder-media-slider.md [hunk 19]
```diff
@@ -442,13 +442,15 @@ If you want a title above the slider, simply add a Text content type above the s
...
+| Infinite Loop  | Determines if the slide show replays from the beginning after the last slide. Options: Yes - The slide show replays indefinitely while the page is open. No - The slide show plays only once.                                                                                                                            |
```
Comment (hickskenh) [inline src/cms/page-builder-media-slider.md]:
```suggestion
| Infinite Loop  | Determines if the slide show replays from the beginning after the last slide. Options: <br/>**Yes** - The slide show replays indefinitely while the page is open. <br/>**No** - The slide show plays only once.                                                                                                                            |
```

File: src/cms/page-builder-media-slider.md [hunk 20]
```diff
@@ -442,13 +442,15 @@ If you want a title above the slider, simply add a Text content type above the s
...
+| Show Arrows    | Determines if each slide includes _next_ and _previous_ navigation arrows on the left and right side of each slide. Options: Yes - Navigation arrows appear on the left and right side of each slide. No - The slides do not have navigation arrows.                                                                     |
```
Comment (hickskenh) [inline src/cms/page-builder-media-slider.md]:
```suggestion
| Show Arrows    | Determines if each slide includes _next_ and _previous_ navigation arrows on the left and right side of each slide. Options: <br/>**Yes** - Navigation arrows appear on the left and right side of each slide. <br/>**No** - The slides do not have navigation arrows.                                                                     |
```

File: src/cms/page-builder-media-slider.md [hunk 21]
```diff
@@ -442,13 +442,15 @@ If you want a title above the slider, simply add a Text content type above the s
...
+| Show Dots      | Determines if the slider includes navigation dots below the slides. Options: Yes - Navigation dots appear at the bottom of the slider. No - The slider does not include navigation dots.                                                                                                                                 |
```
Comment (hickskenh) [inline src/cms/page-builder-media-slider.md]:
```suggestion
| Show Dots      | Determines if the slider includes navigation dots below the slides. Options: <br/>**Yes** - Navigation dots appear at the bottom of the slider. <br/>**No** - The slider does not include navigation dots.                                                                                                                                 |
```

File: src/cms/adobe-stock-manage.md (hunk 2 above)
Comment (leslietilling) [reply src/cms/adobe-stock-manage.md]: it's a github display thing

File: src/cms/adobe-stock-manage.md (hunk 3 above)
Comment (leslietilling) [reply src/cms/adobe-stock-manage.md]: doesn't affect output, but we will fix later

File: src/cms/adobe-stock-manage.md (hunk 4 above)
Comment (leslietilling) [reply src/cms/adobe-stock-manage.md]: doesn't affect output, but we will fix later

File: src/cms/adobe-stock-manage.md (hunk 5 above)
Comment (leslietilling) [reply src/cms/adobe-stock-manage.md]: same

File: src/cms/adobe-stock-manage.md (hunk 6 above)
Comment (leslietilling) [reply src/cms/adobe-stock-manage.md]: same

File: src/cms/adobe-stock.md (hunk 7 above)
Comment (leslietilling) [reply src/cms/adobe-stock.md]: same

File: src/cms/adobe-stock.md (hunk 8 above)
Comment (leslietilling) [reply src/cms/adobe-stock.md]: same

**Comment by leslietilling** [comment]
running tests

**Comment by hickskenh** [comment]
running tests

**Pull request merged**